.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@SteveL-MSFT, your last commit had 5 failures in `PowerShell-CI-macos`
Invoke-WebRequest tests.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: cb4eb0c12c
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.155.1/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Invoke-RestMethod tests.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: 0644804b3d
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.155.1/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Validate Invoke-WebRequest and Invoke-RestMethod -InFile.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: a9e3eb29ae
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.155.1/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Connection.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: 397c12d4b8
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.155.1/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Web cmdlets tests using the cmdlet's aliases.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: 66a3eec27a
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.155.1/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```